### PR TITLE
SynTags shapes2d, masks2d, masks3d, partitions

### PR DESCRIPTION
**Pull request by RAMilewski**

### CODE REVIEW

File: .github/workflows/gen_docs.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     - name: Clone Wiki
       uses: actions/checkout@v3
       with:
-        repository: revarbat/BOSL2.wiki
+        repository: RAMilewski/BOSL2.wiki
```
Comment (revarbat) [inline .github/workflows/gen_docs.yml:14]: This shouldn't be part of a PR into mainline.

File: .github/workflows/gen_tutorials.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ jobs:
     - name: Clone Wiki
       uses: actions/checkout@v3
       with:
-        repository: revarbat/BOSL2.wiki
+        repository: RAMilewski/BOSL2.wiki
```
Comment (revarbat) [inline .github/workflows/gen_tutorials.yml:14]: This also shouldn't be part of a PR into mainline.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@ jobs:
     - name: Clone Wiki
       uses: actions/checkout@v3
       with:
-        repository: revarbat/BOSL2.wiki
+        repository: RAMilewski/BOSL2.wiki
```
Comment (revarbat) [inline .github/workflows/main.yml:36]: This shouldn't be part of a PR into mainline.

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -74,7 +74,7 @@ jobs:
...
```
Comment (revarbat) [inline .github/workflows/main.yml:77]: This shouldn't be part of a PR into mainline.

File: .github/workflows/gen_docs.yml (hunk 1 above)
Comment (RAMilewski) [reply .github/workflows/gen_docs.yml:14]: Sorry.  I was trying to generate the wiki in my repository.  Need to sharpen my github skills.  I'll have to figure out how to exempt those files from the P/R.

**Pull request closed without merging**